tool/lib/commands/shared.dart

```diff
@@ -41,12 +41,12 @@ extension BuildCommandArgsExtension on ArgParser {
     );
   }
 
-  void addUseFlutterFromPathFlag() {
+  void addUpdateFlutterFlag() {
     addFlag(
-      BuildCommandArgs.useFlutterFromPath.flagName,
-      negatable: false,
-      defaultsTo: false,
-      help: 'Whether to use the Flutter SDK on PATH instead of the Flutter SDK '
+      BuildCommandArgs.updateFlutter.flagName,
+      negatable: true,
+      defaultsTo: true,
+      help: 'Whether to update the Flutter SDK before building DevTools '
```

Comment:
nit: remove "before building DevTools" because this flag can be used for more than just the build command.

After removing, it should say 'Whether to update the Flutter SDK contained in the "tool/flutter-sdk" directory'

Reply:
Done!

+ another minor fix I forgot to push earlier, which ensures if you pass `--no-update-flutter` we remove it before passing on to the local server.